### ISSUE TITLE
[BUG]  Accessibility of Pathfinder "feature" box ::hover state 

### ISSUE THREAD

**Issue by tblundell** [unimelb/design-system-issues#30]
Just thought I'd query the accessibility of the pathfinder box with class="featured" when in the :hover state. The background, text and button are all very similar. This issue should only be applicable to desktops as Mob/touch devices don't really make use of hover.

## Expected behaviour
On ::hover the content should ideally be more legible. 

## Current behaviour
Button and text have low contrast when compared to the background colour. 
Screenshot taken from https://lms.unimelb.edu.au/canvas
 
![pathfinders-hover](https://user-images.githubusercontent.com/943446/69194928-81d18c00-0b7e-11ea-9664-799ef4d5b66a.gif)

## Steps to reproduce
1. Set up pathfinder with an item having class="featured" added to `<a> `
2. View page, and hover mouse over the "featured" pathfinder block.

### Affected browsers

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] IE 11
- [X ] Safari 8+
- [ ] iOS 8.4+
- [ ] Android 4.4+

## Possible solution

Update design (& CSS) to more contrasting colours.

**Comment by jaredh-um** [comment]
It looks like the 'featured' class doesn't work optimally with the white and soft pathfinder types. The below needs to be repeated for the 3 and 4 type pathfinders also.

- [ ] Amend `.uomcontent [role="main"] .pathfinder-2.white a.featured p, .uomcontent [role="main"] .pathfinder-2.white a.featured span.text, .uomcontent [role="main"] .pathfinder-2.white a.featured strong` to have `color` white

- [ ] Amend `.uomcontent [role="main"] .pathfinder-2.soft a.featured p, .uomcontent [role="main"] .pathfinder-2.soft a.featured span.text, .uomcontent [role="main"] .pathfinder-2.soft a.featured strong` to have `color` white

- [ ] Amend `.uomcontent [role="main"] .pathfinder-2 a.featured` to have `background-color` Blue Mid 2

- [ ] Amend `.uomcontent [role="main"] .pathfinder-2 a.featured:focus, .uomcontent [role="main"] .pathfinder-2 a.featured:hover` `background-color` to be Blue Dark

**Comment by tomstringer** [comment]
I think this was fixed with unimelb/gen3#142
